content/F2015/F15-event-orientation.md

```diff
@@ -1,5 +1,4 @@
-title: Orientation Event and Dinner Date: 2015-09-13 15:00-18:00 Category: Events Tags: social, big csters, academic Slug: orientation-F15 Author: Fatema Boxwala Summary: Come out and learn about the university life, and get to know some
-awesome women in CS!
+title: Orientation Event and Dinner Date: 2015-09-13 15:00-18:00 Category: Events Tags: social, big csters, academic Slug: orientation-F15 Author: Fatema Boxwala Summary: Come out and learn about the university life, and get to know some awesome women in CS!
```

Comment:
Ahh, sorry! I must have confused you.

Each one of these metadata entries must have its own line, like 
```
key: value can be really long etc.
```

So it's just the `Summary` that needs to be contained on one line, not all the metadata. If you can fix this today, let me know, otherwise I'll close this and make the fix.